stdpopsim/genetic_maps.py

```diff
@@ -169,12 +169,24 @@ def download(self):
             logger.info("Storing map in {}".format(self.map_cache_dir))
             shutil.move(extract_dir, self.map_cache_dir)
 
+    def contains_chromosome_map(self, name):
+        """
+        Just a quick check to see whether of not
+        this genetic map contains a genetic map for `name`
+        """
+        map_file = os.path.join(self.map_cache_dir, self.file_pattern.format(name=name))
+        if not os.path.exists(map_file):
```

Comment:
``return os.path.exists(map_file)`` is more concise and clearer

Also, the doc string comment should describe what the method does: Returns True if this genetic map contains a chromosome with the specified name.